contributing_to_docs/doc_guidelines.adoc

```diff
@@ -392,6 +392,18 @@ In the example, `cloud-provider-first` and `cloud-provider` are not defined by t
 For more information about creating text snippets, see the
 link:https://redhat-documentation.github.io/modular-docs/#using-text-snippets[Red Hat modular docs reference guide]
 
+[id="Auto-generated-content"]
+== Auto-generated content
+
+* The following content is auto-generated in each release and should not be manually edited:
```

Comment:
```suggestion
* The following content is auto-generated in each release and must not be manually edited:
```
I also suggest make this a statement, not a list item, and changing the others to first-level list items.